Lib/distutils/util.py

```diff
@@ -419,8 +420,10 @@ def byte_compile (py_files,
              direct=1)
 """ % (optimize, force, prefix, base_dir, verbose))
 
+        msg = distutils._DEPRECATION_MESSAGE
         cmd = [sys.executable]
         cmd.extend(subprocess._optim_args_from_interpreter_flags())
+        cmd.append(f'-Wignore:{msg}:DeprecationWarning')
```

Comment:
Sadly, -W option escapes the message using re.escape(), it's not possible to use a regex :-(